subprojects/file-watching/src/main/java/org/gradle/internal/watch/vfs/BuildLifecycleAwareVirtualFileSystem.java

```diff
@@ -32,7 +32,12 @@
     /**
      * Called when the build is started.
      */
-    void afterBuildStarted(boolean watchingEnabled, VfsLogging vfsLogging, WatchLogging watchLogging, BuildOperationRunner buildOperationRunner);
+    void afterBuildStarted(WatchMode watchingEnabled, VfsLogging vfsLogging, WatchLogging watchLogging, BuildOperationRunner buildOperationRunner);
+
+    /**
+     * Returns whether watching the file system is supported on the current operating system, and has successfully been started.
```

Comment:
I think this Javadoc is wrong: It currently returns whether
- watching the file system is supported and
- watching the file has been enabled
- and it has been successfully started and is currently watching

Reply:
Clarified.